science/2022-11-23-science-blog-6/index_de.md

```diff
@@ -0,0 +1,237 @@
+
+<img src="./rki_covid_lk_inz_paired_plot-1.png" class="figure-img img-fluid" alt="Quotient der 7-Tage-Inzidenzen im zeitlichen Verlauf (50 Paare von Kreisen)."/>
+<figcaption aria-hidden="true">Abb. 3: Quotient der 7-Tage-Inzidenzen im zeitlichen Verlauf (50 Paare von Kreisen).</figcaption>
+<br/>
+
+Wir sehen, dass sich die Inzidenzen in den Kreisen nur gering unterschieden, nur in wenigen Ausnahmen wird ein Faktor 10 erreicht. Eine so starke Beeinflussung der unterschiedlichen Positivenanteile aufgrund lokaler Unterschiede in der Inzidenz, wie in obigem Beispiel beschriebenen, kann damit ausgeschlossen werden.
+
+## 4 Positivenanteile (lokal)
+
+Wir werden im Folgenden die lokalen Positivenanteile untersuchen. Dabei betrachten wir zunächst Antigen-Schnelltests (RAT) und PCR-Tests gemeinsam. Da diese beiden Gruppen, wie eingangs beschrieben und erläutert, sehr unterschiedliche Positivenanteile haben, hängen die Gesamt-Positivenanteile auch von der relativen Häufigkeit beider Testarten ab. Die folgende Abbildung zeigt den Positivenanteil im zeitlichen Verlauf, gemittelt über alle Kreise bzw. bundesweit.
+
+<img src="./ppa_test_result_lk_smooth_all-1.png" class="figure-img img-fluid" alt="Positivenanteil nach Risikobenachrichtigung im zeitlichen Verlauf, geglättet und mit Konfidenzintervall, gemittelt über alle Kreise (dick) bzw. bundesweit (dünn)."/>
+<figcaption aria-hidden="true">Abb. 4: Positivenanteil nach Risikobenachrichtigung im zeitlichen Verlauf, geglättet und mit Konfidenzintervall, gemittelt über alle Kreise (dick) bzw. bundesweit (dünn).</figcaption>
+<br/>
+
+Der Positivenanteil nach einer roten Warnung ist stets größer als nach einer grünen Warnung (niedriges Risiko mit Risikobegegnung). Bei hohen Inzidenzen ist der Positivenanteil nach niedrigem Risiko ohne Risikobegegnung gelegentlich größer und gleichzeitig stärker von der Inzidenz abhängig (steigt und fällt schneller) als nach einer roten Warnung. Eine getrennte Betrachtung von Antigen-Schnelltests und PCR-Tests kann hier erhellen.
+
+Um einen Eindruck der lokalen Unterschiede zu geben, sind in der folgenden Abbildung die Verläufe für vier zufällig gewählte Kreise gezeigt.
+
+<img src="./ppa_test_result_lk_smooth_sample-1.png" class="figure-img img-fluid" alt="Positivenanteil nach Risikobenachrichtigung im zeitlichen Verlauf, geglättet und mit Konfidenzintervall, für vier zufällig gewählte Kreise."/>
+<figcaption aria-hidden="true">Abb. 5: Positivenanteil nach Risikobenachrichtigung im zeitlichen Verlauf, geglättet und mit Konfidenzintervall, für vier zufällig gewählte Kreise.</figcaption>
+<br/>
+
+Auch hier sehen wir deutliche die größere Abhängigkeit des Positivenanteils bei niedrigem Risiko ohne Risikobegegnung von der Inzidenz deutlich: die grüne Kurve schwankt stärker als die rote oder gelbe.
```

Comment:
There is something wrong with this sentence in the use of "deutliche" / "deutlich". I don't know what it is supposed to be.

Reply:
thanks - fixed it!